assignment.md

```diff
@@ -1,23 +1,31 @@
-What is the difference between push, pull, and fetch?
+# Important Git commands to understand 
 
-# Understanding git commands push, pull, and fetch
+To assist you in understanding Git concepts and related commands, Git command definitions are provided below.
 
-**`git push`** - pushes changes from your local branch to a remote repository.
+**`git push`** - Pushes changes from your local branch to a remote repository.
 
-**`git fetch`** - fetches changes from a remote repository into the tracking branch.
+**`git fetch`** - Fetches changes from a remote repository into the tracking branch.
 
-**`git pull`** - pulls the changes from a remote branch into the tracking branch and merges the changes into your local branch. 
+**`git pull`** - Pulls changes from a remote branch into the tracking branch and merges the changes into your local branch. 
```

Comment:
Capitalized the first character.   Some companies I have worked at make the first character lower case.  What do you think ?